### DISCUSSION TITLE
VSD for Adults

### DISCUSSION THREAD

**Discussion by sagostine** [UNC-Project-Open-AAC/OS-DPI#172] [Q&A] [answered]
Hi all!

I am attempting to make a VSD geared towards adults this week. I have gotten pretty close to what I initially envisioned but am now stuck. I was able to create tabs that displayed 4 picture specific comments and I am able to toggle through them. However, when I click on the different tabs the pictures don't change anymore. Any thoughts on how I can fix this? Ideally, I would love it for the person to just be able to click on the tab instead of the VSD button and then the tab! I've attached my excel sheet. I can't see to attach the OSDPI file so let me know and I can send it to you!

Thanks!
Susie 
[Luna_VSD_Adult (9).xlsx](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/10925223/Luna_VSD_Adult.9.xlsx)

**Comment by lageist** [comment]
Hi Susie @sagostine! I'd love to take a look and try to figure this out with you. GitHub doesn't recognize the .osdpi extension, but it's worked for me to compress to a .zip file and attach that here.

**Comment by sofia-bg** [comment]
@sagostine I get around that by uploading files to the "Designs in Progress" folder in the <>Code page. 
You select "Designs in Progress" -->> upload files -->> commit changes 

Once uploaded, select the file you uploaded (may have bounced you out and may need to go back into "Designs in Progress" 
It will take you to a new page that doesn't look like much is there - but it allows others to download the file you uploaded. I usually go to the button with ... and then select "copy permalink" you can then paste that link into your comment.

**Reply by sofia-bg** [reply]
or what @lageist says, I suppose not necessarily a design in progress that the whole team needs, but I think it is nice to have things saved there so others can poke around and see types of problems/solutions others have worked through

**Comment by gbishop** [comment]
A little known fact is that .os-dpi files are really just .zip files with
the suffix changed. You can simply change the suffix from .os-dpi to .zip
and then you can upload it. A .zip file is importable into our app.

**Reply by sofia-bg** [reply]
@gbishop was just about to email you. I changed an osdpi file of mine to .zip and tried to import and got an error message that it could not read something about it among other things...

**Comment by sagostine** [comment]
[Luna_VSD_Adult_WED.zip](https://github.com/UNC-Project-Open-AAC/OS-DPI/files/10944773/Luna_VSD_Adult_WED.zip)


Thanks for the suggestion, I was able to change the suffix and (hopefully) you all can see the file now! I plan to mess around with it a bit more this afternoon and will update if I figure anything out!

**Comment by sagostine** [comment, marked as answer]
@lageist I figured it out! I just had to add the VSD into the tab panel instead of it being its own separate thing!

**Reply by aetkie** [reply]
Great job figuring it out @sagostine !

**Reply by sofia-bg** [reply]
@sagostine adding tags and wondering if you can share the working draft so that if others come to this thread they can see the solution you arrived at! thanks :)

**Comment by gbishop** [comment]
Glad to hear it!
